DogsList/DogsList/Controllers/SearchStarWarViewController.swift

```diff
@@ -67,7 +75,7 @@ extension SearchStarWarViewController {
     private func alertGeenrate(alertTitle: String, alertMessage: String, actionTitle: String, handler: ((UIAlertAction) -> Void)? = nil) {
         let alert = UIAlertController(title: alertTitle, message: alertMessage, preferredStyle: .alert)
         alert.addAction(UIAlertAction(title: actionTitle, style: .default) {(_: UIAlertAction!) in
-            self.viewDidLoad()})
+                            self.viewDidLoad()})
```

Comment:
add SwiftLint to the project and fix all errors/warnings

Reply:
Ok. SwiftLint has already been added.